website_sale_qty/static/src/js/website_sale_qty.js

```diff
@@ -0,0 +1,47 @@
+/**
+*    Copyright 2016 LasLabs Inc.
+*    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+**/
+
+odoo.define('website_sale_qty', function(require) {
+    'use strict';
+
```

Comment:
`var $ = require('$')` - even though it works, you haven't guaranteed that jQuery exists in the pipeline before using in L19&20